frontend-instructor/src/App.jsx

```
@@ -0,0 +1,18 @@
+import "./App.css";
+import Footer from "./components/footer/Footer";
+import NavBar from "./components/navBar/NavBar";
+import { Outlet } from "react-router-dom";
+
+function App() {
+  return (
+    <div className="h-screen flex flex-col">
+      <NavBar />
+      <div className="grow shrink-0 basis-auto">
+        <Outlet></Outlet>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)